modules/infinite-scroll/infinity.php

```diff
@@ -372,11 +372,31 @@ function settings_api_init() {
 		if ( ! current_theme_supports( 'infinite-scroll' ) )
 			return;
 
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			// This setting is no longer configurable in wp-admin on WordPress.com -- leave a pointer
+			add_settings_field( self::$option_name_enabled,
+				'<span id="infinite-scroll-options">' . esc_html__( 'Infinite Scroll Behavior', 'jetpack' ) . '</span>',
+				array( $this, 'infinite_setting_html_calypso_placeholder' ),
+				'reading'
+			);
+			return;
+		}
+
 		// Add the setting field [infinite_scroll] and place it in Settings > Reading
 		add_settings_field( self::$option_name_enabled, '<span id="infinite-scroll-options">' . esc_html__( 'Infinite Scroll Behavior', 'jetpack' ) . '</span>', array( $this, 'infinite_setting_html' ), 'reading' );
 		register_setting( 'reading', self::$option_name_enabled, 'esc_attr' );
 	}
 
+	function infinite_setting_html_calypso_placeholder() {
+		$details = get_blog_details();
+		echo '<span>' . sprintf(
+			/* translators: Variables are the enclosing link to the settings page */
+			esc_html__( 'This option has moved. You can now manage it %1$shere%2$s.' ),
+			'<a href="' . esc_url( 'https://wordpress.com/settings/writing/' . $details->domain ) . '">',
```

Comment:
This is not a breaking change since that field is not displayed on Jetpack sites, but it is worth noting for later that `$details->domain` won't work for all Jetpack sites. It will be an issue for sites that are in a subdirectory. It may be worth switching to something like `Jetpack::build_raw_urls( get_home_url() );` to solve this.